Documentation/developer-guide.md

```diff
@@ -30,61 +27,48 @@ In order to build .NET Command Line Interface, you need the following installed
 
 ## Building/Running
 
-1. Run `build.cmd` or `build.sh` from the root depending on your OS. If you don't want to execute tests, run `build.cmd -Targets Prepare,Compile` or `./build.sh --targets Prepare,Compile`.
-2. Use `artifacts/{os}-{arch}/stage2/dotnet` to try out the `dotnet` command. You can also add `artifacts/{os}-{arch}/stage2` to the PATH if you want to run `dotnet` from anywhere.
+1. Run `build.cmd` or `build.sh` from the root depending on your OS. If you don't want to execute tests, run `build.cmd /t:Compile` or `./build.sh /t:Compile`. 
+2. Use `artifacts/{RID}/stage2/dotnet` to try out the `dotnet` command. You can also add `artifacts/{os}-{arch}/stage2` to the PATH if you want to run `dotnet` from anywhere.
 
 ## A simple test
 
-1. `cd TestAssets\TestProjects\TestSimpleIncrementalApp`
-2. `dotnet restore`
-3. `dotnet run`
+1. `cd {new directory}`
```

Comment:
This guide is for developers who want to work in the CLI repo or extend the CLI in their own repos. I'm sure there is a bunch of missing content here, so please do suggest any additions you think we need. As @mishra14 pointed out in the source issue, this document has been neglected.